examples/simpletreeview/dub.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "simpletreeview",
+    "description": "Example of TreeView using qml.",
+    "copyright": "Copyright © 2016-2017, Filippo Cucchetto, Alexandr Druzhinin",
+    "authors": ["Filippo Cucchetto, Alexandr Druzhinin"],
+    "targetPath": "bin",
+    "copyFiles": ["source/app.qml"],
```

Comment:
Add default.txt do copyFiles otherwise the application throws an exception and exit :)